kafka-admin/src/main/java/org/bf2/admin/kafka/admin/handlers/CommonHandler.java

```diff
@@ -12,6 +12,7 @@
 import org.apache.kafka.common.errors.InvalidTopicException;
 import org.apache.kafka.common.errors.LeaderNotAvailableException;
 import org.apache.kafka.common.errors.PolicyViolationException;
+import org.apache.kafka.common.errors.ResourceNotFoundException;
```

Comment:
@biswassri , the exception to look for is `javax.ws.rs.NotFoundException`. A test like the following in `RestEndpointTestIT` demonstrates the issue (will fail on `main` branch).

```java
    @Test
    void testUnknownEndpointReturnsNotFound() {
        given()
            .log().ifValidationFails()
            .when()
                .get("/foo/bar")
            .then()
                .log().ifValidationFails()
                .statusCode(Status.NOT_FOUND.getStatusCode());
    }
```

Reply:
Oh this is the test that used to get skipped on the main. Right! I'll try this

Reply:
@MikeEdgar  I updated the class and tried this test. It passes with these PR changes. Do you want me to commit this `testUnknownEndpointReturnsNotFound()` to the PR as well?

Reply:
> Do you want me to commit this `testUnknownEndpointReturnsNotFound()` to the PR as well?

Yes, please. Sorry, I thought I had answered you on this.

Reply:
No worries Mike! I'll go ahead and add this in.